webhook/webhook.go

```diff
@@ -326,7 +327,7 @@ func (ac *AdmissionController) register(
 	var rules []admissionregistrationv1beta1.RuleWithOperations
 	for gvk := range ac.Handlers {
 		// Lousy pluralizer
-		plural := strings.ToLower(gvk.Kind) + "s"
+		plural := strings.ToLower(inflect.Pluralize(gvk.Kind))
```

Comment:
Do we know if this will change existing behavior of other types?

Reply:
The tool has a set of rules for plural conversion “smartly”.
We didn't hit this issue before because all of the existing resources did not need "es" as suffix when converting to plural.

In fact I borrow this logic from [kubebuilder](https://github.com/kubernetes-sigs/kubebuilder/blob/master/pkg/scaffold/resource/crd.go#L56). so k8s api should follow the same rules. And we won't break backwards compatibility.

Reply:
Can you get rid of the comment, since this is no longer a "Lousy pluralizer" :)

Reply:
done